### ISSUE TITLE
JanusGraphException: Could not call index when doing multithreaded loading

### ISSUE THREAD

**Issue by shanghai-Jerry** [graknlabs/grakn#5402]
## Description

Using grakn java client

## Environment

1. OS (where Grakn server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
    Ubuntu 16.4
2. Grakn version (and platform): { e.g. Grakn Core 1.4, or Grakn KGMS 1.4.3 on Google Cloud }
    Grakn Core 1.5.7
3. Grakn client: { e.g. client-java, client-python, client-nodejs, or console }
    console
4. Other environment details:
   Nope

## Reproducible Steps

Steps to create the smallest reproducible scenario:
Using  grakn client in java

```java
int count = 0;
    int finished = 0;
    GraknClient.Session session = hgraknClient.NewSession(hgraknClient.getKeySpace());
    GraknClient.Transaction transaction = session.transaction().write();
    try {
      for (JsonObject item : this.data) {
        count++;
        finished++;
        GraqlQuery graqlInsertQuery = input.template(item);
        transaction.execute(graqlInsertQuery);

        // one by one commit
        // transaction.commit();

        if (count % 1000 == 0) {
          logger.info(Thread.currentThread().getName()+" - Executing Graql Insert: " +
              finished + "/" + this.data.size());
          transaction.commit();
          transaction = session.transaction().write();
          count = 0;
        }
      }
      if (count > 0) {
          transaction.commit();
      }
    } catch (Exception e) {
      logger.info("[commit error] => " + e.getMessage());
    }
```

it's running in parallel with 20 threads, with about 4000w concepts, each single thread inserts with 1000 concepts.
But it's ok when running with total dataset in single thread. 

## Expected Output
success
## Actual Output
[commit error] => UNKNOWN: Could not call index. Please check server logs for the stack trace
[commit error] => null

## Additional information
```java
  2019-08-27 09:57:37,228 [transaction-listener-0] ERROR g.c.s.r.SessionService$TransactionListener - Runtime Exception in RPC TransactionListener:
org.janusgraph.core.JanusGraphException: Could not call index
	at org.janusgraph.graphdb.util.SubqueryIterator.<init>(SubqueryIterator.java:68)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx$3.execute(StandardJanusGraphTx.java:1298)
	at org.janusgraph.graphdb.transaction.StandardJanusGraphTx$3.execute(StandardJanusGraphTx.java:1190)
	at org.janusgraph.graphdb.query.QueryProcessor$LimitAdjustingIterator.getNewIterator(QueryProcessor.java:194)
	at org.janusgraph.graphdb.query.LimitAdjustingIterator.hasNext(LimitAdjustingIterator.java:68)
	at com.google.common.collect.Iterators$5.computeNext(Iterators.java:662)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:145)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:140)
	at org.janusgraph.graphdb.query.ResultSetIterator.nextInternal(ResultSetIterator.java:54)
	at org.janusgraph.graphdb.query.ResultSetIterator.<init>(ResultSetIterator.java:44)
	at org.janusgraph.graphdb.query.QueryProcessor.iterator(QueryProcessor.java:66)
	at com.google.common.collect.Iterables$4.iterator(Iterables.java:578)
	at org.janusgraph.graphdb.tinkerpop.optimize.JanusGraphStep.executeGraphCentryQuery(JanusGraphStep.java:156)
	at org.janusgraph.graphdb.tinkerpop.optimize.JanusGraphStep.lambda$null$1(JanusGraphStep.java:95)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at org.janusgraph.graphdb.tinkerpop.optimize.JanusGraphStep.lambda$new$2(JanusGraphStep.java:95)
	at org.apache.tinkerpop.gremlin.process.traversal.step.map.GraphStep.processNextStart(GraphStep.java:142)
	at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(AbstractStep.java:143)
	at org.apache.tinkerpop.gremlin.process.traversal.util.DefaultTraversal.hasNext(DefaultTraversal.java:192)
	at grakn.core.server.session.TransactionOLTP.getConcept(TransactionOLTP.java:496)
	at grakn.core.server.session.TransactionOLTP.getConcept(TransactionOLTP.java:491)
	at grakn.core.server.kb.concept.AttributeTypeImpl.attributeWithLock(AttributeTypeImpl.java:183)
	at grakn.core.server.kb.concept.AttributeTypeImpl.lambda$putAttribute$3(AttributeTypeImpl.java:129)
	at grakn.core.server.kb.concept.AttributeTypeImpl.putInstance(AttributeTypeImpl.java:141)
	at grakn.core.server.kb.concept.AttributeTypeImpl.putAttribute(AttributeTypeImpl.java:129)
	at grakn.core.server.kb.concept.AttributeTypeImpl.create(AttributeTypeImpl.java:114)
	at grakn.core.graql.executor.ConceptBuilder.putInstance(ConceptBuilder.java:395)
	at grakn.core.graql.executor.ConceptBuilder.tryPutConcept(ConceptBuilder.java:228)
	at grakn.core.graql.executor.ConceptBuilder.build(ConceptBuilder.java:165)
	at grakn.core.graql.executor.WriteExecutor.buildConcept(WriteExecutor.java:291)
	at grakn.core.graql.executor.WriteExecutor.getConcept(WriteExecutor.java:404)
	at grakn.core.graql.executor
```

**Comment by marco-scoppetta** [comment]
Hi @shanghai-Jerry , thanks for reporting the issue, we have managed to reproduce it and working on fixing it

**Comment by marco-scoppetta** [comment]
This should be solved by moving to the new CQL driver which is already in master and available with Grakn 1.6.0
